ansible/eos_install_config

```diff
@@ -107,7 +124,8 @@ def main():
             password=dict(required=True),
             use_ssl=dict(required=False, default=True),
             config_file=dict(required=True),
-            commit_changes=dict(required=True),
+            commit_changes=dict(required=False, default=False),
+            replace_config=dict(required=False, default=False),
```

Comment:
Did anyone notice me changing the default for "commit_changes" to False? :) I figured that would be appropriate for if I forget the option (I would do the same in napalm).